test/state/state.cpp

```diff
@@ -103,8 +103,8 @@ Account& State::get_or_insert(const address& addr, Account account)
 
 Account& State::touch(const address& addr)
 {
-    auto& acc = get_or_insert(addr);
-    if (!acc.erasable)
+    auto& acc = get_or_insert(addr, {.erasable = true});
```

Comment:
What does `erasable` mean?

Reply:
`erasable` is bad name for `touched` or in other words `erase_if_empty`. I plan to change the name to the `erase_if_empty` at some point. Any better names?